composer.lock

```diff
@@ -22620,5 +22620,5 @@
     "platform-overrides": {
         "php": "8.1.12"
     },
-    "plugin-api-version": "2.6.0"
+    "plugin-api-version": "2.3.0"
```

Comment:
is this correct?

Reply:
done in 141177279005fd4a88d0714c1bc1770cf606d785
the style fixer changes seem to be ok...
(CustomerInterface $customer = null) within the interface seems to work as well as (?CustomerInterface $customer)